extensions/librdkafka/ConsumeKafka.cpp

```diff
@@ -211,7 +211,7 @@ void ConsumeKafka::onSchedule(core::ProcessContext* context, core::ProcessSessio
   context->getProperty(DuplicateHeaderHandling.getName(), duplicate_header_handling_);
 
   headers_to_add_as_attributes_ = utils::listFromCommaSeparatedProperty(*context, HeadersToAddAsAttributes.getName());
-  max_poll_records_ = gsl::narrow<std::size_t>(utils::getOptionalUintProperty(*context, MaxPollRecords.getName()).value_or(DEFAULT_MAX_POLL_RECORDS));
+  max_poll_records_ = gsl::narrow<std::size_t>(utils::getOptionalProperty<uint64_t>(*context, MaxPollRecords.getName()).value_or(DEFAULT_MAX_POLL_RECORDS));
```

Comment:
`ProcessContext::getProperty<T>(const Property&)` already returns an `std::optional<T>`, does `getOptionalProperty` behave differently?

Reply:
I wasn't aware of this, in that case this is not needed at all, removed it in ddfa1b976649599300457af5d5b4100f948d6a43